report/3.Theory/LanguageSpecification.tex

```diff
@@ -76,6 +85,27 @@ \section{Language Specification}
 ["john", "frank"] where {value is "frank"} print
 \end{lstlisting}
 
+\textbf{Grammar for function calls}\\
+There are three different ways to call a function depending on how many parameters must be passed with the function:
```

Comment:
maybe explain infix, so something like "first argument is on the left side, while the remaining is on the right".

whole sentence could be restructured to something along the lines of

"Function calls are done with infix notation, meaning that if a given method is defined with parameters, the first argument has to be in front of the function identifier, while all remaining arguments are postfixed.

As explained by the following examples:
- printHelloWorld
- "hello world :)" print
- "hello" printWords ["world",":)"]